src/MetricsTests/Microsoft.ApplicationInsights.Metrics.Core.Test.Intrnl/Microsoft.ApplicationInsights.Metrics.Core.Test.Intrnl.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\NuGet.Packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\..\..\..\NuGet.Packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{0272FE8B-9B25-43B0-A26B-3C71154DEBBA}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.ApplicationInsights.Metrics.Core.Test.Intrnl</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.Metrics.Core.Test.Intrnl</AssemblyName>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
project targeted 4.5. Why tests are on 4.6.2?

Reply:
Good point. Need to fix while moving tests.